cmd/operator-sdk/internal/genutil/genutil.go

```diff
@@ -47,7 +48,8 @@ func runGoBuildCodegen(binDir, repoDir, genDir string) error {
 	}
 
 	// Only print binary build info if verbosity is explicitly set.
-	if projutil.IsGoVerbose() {
+	if viper.GetBool(flags.VerboseOpt) {
+		_ = os.Setenv(projutil.GoFlagsEnv, os.Getenv(projutil.GoFlagsEnv)+" -v")
```

Comment:
If `GOFLAGS="-v"` is set already this will cause an error:

```console
go run: v flag may be set only once
```

I recommend reworking `IsGoVerbose()` into something like:

```go
var flagRe = regexp.MustCompile("(.* )?-v(.* )?")

// SetGoVerbose sets GOFLAGS="${GOFLAGS} -v" if GOFLAGS does not
already contain "-v" to make "go" command output verbose.
func SetGoVerbose() error {
	gf, ok := os.LookupEnv(GoFlagsEnv)
	if !ok || len(gf) == 0 {
		return os.Setenv(GoFlagsEnv, "-v")
	}
	if !flagRe.MatchString(gf) {
		return os.Setenv(GoFlagsEnv, gf+" -v")
	}
	return nil
}
```

And use it here, in [`GoTest()`](https://github.com/operator-framework/operator-sdk/blob/3c8e5e7280d8b8584e46605faba999e4fa3c4487/internal/util/projutil/exec.go#L82), and in [`goCmd()`](https://github.com/operator-framework/operator-sdk/blob/3c8e5e7280d8b8584e46605faba999e4fa3c4487/internal/util/projutil/exec.go#L93), the latter two with the same verbosity check.